### PR TITLE
IS-1203 changes to package.json and package-lock.json

### PR DESCRIPTION
**Pull request by mattraffel**
Updating version syntax

### CODE REVIEW

**Comment by Artemkaaas** [comment]
What is the meaning of these changes?
We still have 3 vulnerabilities (1 low, 1 moderate, 1 high) on `npm i`.

**Comment by mattraffel** [comment]
`npm audit fix` is reporting no no issues, on my local machine.  If we are still having issues then I say the fix I provided is not the solution and we restart from scratch.   The changes I made were **recommended by github**.

**Comment by jovfer** [comment]
@mattraffel @Artemkaaas I suggest you to align npm versions. May be it's a reason why you see different output of npm audit.

File: vcx/wrappers/node/package-lock.json [hunk 1]
```diff
@@ -1,4535 +1,6734 @@
...
-    },
-    "formatio": {
-      "version": "1.2.0",
-      "resolved": "https://registry.npmjs.org/formatio/-/formatio-1.2.0.tgz",
-      "integrity": "sha1-87IWfZBoxGmKjVH092CjmlTYGOs=",
-      "dev": true,
-      "requires": {
-        "samsam": "1.3.0"
-      }
-    }
```
Comment (jovfer) [inline vcx/wrappers/node/package-lock.json]: @mattraffel the github report that it should be 4.0.0+ https://github.com/hyperledger/indy-sdk/network/alert/vcx/wrappers/node/package-lock.json/just-extend/open
but here I can see old version. Could you double check it?

**Comment by jovfer** [comment]
@mattraffel @Artemkaaas my `npm audit` output below (nodejs 8.15.1)
```
sminaev@minaev:~/projects/evernym/indy-sdk/vcx/wrappers/node$ npm audit
                                                                                
                       === npm audit security report ===                        
                                                                                
# Run  npm update handlebars --depth 2  to resolve 1 vulnerability
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ handlebars                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ typedoc [dev]                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ typedoc > handlebars                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/755                       │
└───────────────┴──────────────────────────────────────────────────────────────┘


# Run  npm update nise --depth 2  to resolve 1 vulnerability
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ just-extend                                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ sinon [dev]                                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ sinon > nise > just-extend                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/780                       │
└───────────────┴──────────────────────────────────────────────────────────────┘


┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ braces                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=2.3.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ babel-cli [dev]                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ babel-cli > chokidar > anymatch > micromatch > braces        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/786                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 3 vulnerabilities (1 low, 1 moderate, 1 high) in 7683 scanned packages
  run `npm audit fix` to fix 2 of them.
  1 vulnerability requires manual review. See the full report for details.
```

File: vcx/wrappers/node/package-lock.json (hunk 1 above)
Comment (mattraffel) [reply vcx/wrappers/node/package-lock.json]: Cannot control the package-lock.json file.  its generated.  why npm chose that version, no idea.  We either accept it or don't.  That's the only choices.

**Comment by jovfer** [comment]
@Artemkaaas @mattraffel I've updated this PR by changes from "npm audit fix" from my env. Let's see will it pass CI or not...

File: vcx/wrappers/node/package-lock.json [hunk 2]
```diff
@@ -1,4735 +1,4770 @@
...
-        "ref-struct": "1.1.0"
-      },
-      "dependencies": {
-        "bindings": {
-          "version": "1.2.1",
-          "resolved": "https://registry.npmjs.org/bindings/-/bindings-1.2.1.tgz",
-          "integrity": "sha1-FK1hE4EtLTfXLme0ystLtyZQXxE="
+        "ansi-regex": {
+            "version": "2.1.1",
+            "res
```
Comment (jovfer) [inline vcx/wrappers/node/package-lock.json]: My npm audit has incremented this version, so I hope it will resolve GH warnings

**Pull request merged**